### DISCUSSION TITLE
How to detect a pinch within an element within canvas

### DISCUSSION THREAD

**Discussion by Rajdeepc** [pmndrs/use-gesture#503] [Q&A]
@dbismut 
Currently I am building a feature with onPinch which when user pinches on the canvas it will fire canvas.poinToZoom. But with this I also want to implement element resizing on pinching on each individual component.

This is my current implementation which only zoom to point.

How can i have both these features with onPinch, if I can detect a pinch on an element instead on the blank canvas my job will be done.

**Comment by dbismut** [comment]
You forgot to link to your initial implementation @Rajdeepc

**Reply by Rajdeepc** [reply]
You can close this since we are discussing this on another thread. Thanks
